### PR TITLE
reopening #279

### PR DESCRIPTION
**Pull request by floweisshardt**

### CODE REVIEW

**Comment by fmessmer** [comment]
follow-up PR of #279 due to merge conflicts
the remaining issues have been fixed in the latest commits....

**Comment by fmessmer** [comment]
requires https://github.com/ipa320/cob_common/pull/147

File: cob_hardware_config/raw3-1/config/laser_front.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ port: /dev/ttyScan0
 baud: 500000
 scan_duration: 0.025 #no info about that in SICK-docu, but 0.025 is believable and looks good in rviz
 scan_cycle_time: 0.040 #SICK-docu says S300 scans every 40ms
-inverted: true
+inverted: false
```
Comment (floweisshardt) [inline cob_hardware_config/raw3-1/config/laser_front.yaml:5]: was this change done by purpose?

Comment (floweisshardt) [inline cob_hardware_config/raw3-1/config/laser_front.yaml:5]: @ipa-fxm got this from the robot user on raw3-1

**Comment by floweisshardt** [comment]
visual review: Ok

**Pull request merged**